modular_skyrat/master_files/code/modules/clothing/under/jobs/command.dm

```diff
@@ -0,0 +1,124 @@
+/obj/item/clothing/under/rank/captain
+	worn_icon_digi = 'modular_skyrat/master_files/icons/mob/clothing/under/command_digi.dmi' //Anything that was in TG's captain.dmi, should be in our command_digi.dmi
+	//NOTE - TG uses "captain.dmi"; because we have a few non-captain items going in here for ease of access, this will just be "command.dmi"
+
+/obj/item/clothing/under/rank/captain/skyrat
+	icon = 'modular_skyrat/master_files/icons/obj/clothing/under/command.dmi'
+	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/under/command.dmi'
+
+/////////////////////////////////////
+////////////// CAPTAIN //////////////
+
+/obj/item/clothing/under/rank/captain/skyrat/humble
+	desc = "It's a blue jumpsuit with some small gold markings denoting the rank of \"Captain\", more humble than it's sister."
+	name = "captain's humble jumpsuit"
+	icon_state = "captainhumble"
+
+/obj/item/clothing/under/rank/captain/skyrat/kilt
+	desc = "Not a skirt, it is, however, armoured and decorated with a tartan sash."
+	name = "captain's kilt"
+	icon_state = "capkilt"
+	inhand_icon_state = "kilt"
+
+//Donor item for Gandalf - all donors have access
+/obj/item/clothing/under/rank/captain/skyrat/black
+	name = "captain's black suit"
+	desc = "A very sleek, albeit outdated, naval captain's uniform for those who think they're commanding a battleship."
+	icon_state = "captainblacksuit"
+	inhand_icon_state = "w_suit"
+	can_adjust = FALSE
+
+////////////////////////////////////////
+////////////// BLUESHIELD //////////////
+
+//Why is this in command.dm? Simple: Centcomm.dm will already be packed with CC/NTNavy/AD/LL/SOL/FTU - all of them more event-based clothes, while this will appear
```

Comment:
```suggestion
//Why is this in command.dm? Simple: Centcomm.dmi will already be packed with CC/NTNavy/AD/LL/SOL/FTU - all of them more event-based clothes, while this will appear
```
typo